config/fcos/v1_6_exp/translate.go

```diff
@@ -96,7 +96,7 @@ func (c Config) ToIgn3_5(options common.TranslateOptions) (types.Config, report.
 	return cfg.(types.Config), r, err
 }
 
-// ToIgn3_5Bytes translates from a v1.5 Butane config to a v3.4.0 Ignition config. It returns a report of any errors or
+// ToIgn3_5Bytes translates from a v1.6 Butane config to a v3.5.0 Ignition config. It returns a report of any errors or
```

Comment:
This and the next commit could be folded into their original commits.  Alternatively (and perhaps easier) this commit could be merged with the previous and next ones.

Reply:
Ah, yeah, I did go ahead and refold the comments for less commits into their original commits.   

I did keep the latest commit which was the registration of the bits. I think it makes sense to stay how it is.